### ISSUE TITLE
Just want to know is it been actively developed, since i dont see any recent commits on core or other projects?...

### ISSUE THREAD

**Issue by prabaharanit** [walkmod/walkmod-core#83]

**Comment by rpau** [comment]
Yes,

A big refactor is needed, but it is actively maintained.

**Comment by cr4igo** [comment]
Is it still being maintained by someone? I like the idea behind walkmod. I think about using walkmod for a custom language (not java). What are your plans? No problem, if it needs a big refactoring which does not fit your current time table - maybe my colleagues could help out for the refactoring.

**Comment by octopatch** [comment]
Yes, it is maintained by myself. The reason of the lack of activity is the
lack of community. We can try to meet to understand your needs and how can
I help. There might be other paths that you want to explore.

Please, reach to me using LinkedIn and we schedule a meeting
